astropy/units/__init__.py

```diff
@@ -45,3 +45,20 @@
 # Imperial units.
 
 set_enabled_units([si, cgs, astrophys, function_units, misc, photometric])
+
+
+# -------------------------------------------------------------------------
+
+def __getattr__(name):
+    if name == "littleh":
+        from astropy.units.astrophys import littleh
+        return littleh
+    elif name == "with_H0":
+        from astropy.units.equivalencies import with_H0
+        return with_H0
+
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}.")
+
+
+def __dir__():
+    return list(sorted(set(globals().keys()) | {"littleh", "with_H0"}))
```

Comment:
`sorted` produces a list (same for other `__dir__`)

Reply:
Thanks! Done.